### ISSUE TITLE
Upgrade to datatables.net@1.11.3 in distribution repos

### ISSUE THREAD

**Issue by jonathanloos** [DataTables/DataTablesSrc#195]
`yarn audit` returning XSS vulnerability ([https://snyk.io/vuln/SNYK-JS-DATATABLESNET-1540544](url)) for `datatables.net < 1.11.3`. I'd like to propose an update to the distribution repositories to:

1. `datatables.net-bs4` to point to `datatables.net@1.11.3`.
2. All other distribution repos point to updated version of `datatables.net-bs4`.

If accepted this will be my first contribution to an open source project so please bear with me. If I missed anything here please let me know!

Best, 
Jon

**Comment by AllanJard** [comment]
Good point, we've just assumed that the dependency of `>=` as is at the moment is good enough. That isn't the case for something such as this.

We have scripts which can update all of the distribution repos if you don't fancy changing them all, but if you do, it will be a lot of contributions to get you started in the OSS world :-).

**Comment by jonathanloos** [comment]
Hey @AllanJard ! Wow that was fast.. 

Although I do appreciate the exposure value for me of spraying PRs around I don't think the juice is worth the squeeze. If you guys have a tool to automate this by all means go ahead!

-Jon

**Comment by AllanJard** [comment]
Yup - we'll get it done :)

**Comment by AllanJard** [comment]
We'll hopefully get the changes made today, but actually tagging and releasing everything might take a little while as this would be the only change.

**Comment by jonathanloos** [comment]
Great, thank you!

**Comment by jonathanloos** [comment]
Great, thank you!

**Comment by AllanJard** [comment]
Forgot to close this at the time. Doing so now.
